### ISSUE TITLE
Using pop-up window?

### ISSUE THREAD

**Issue by ssskkkky** [guanyilun/org-elp#5]
I found a interesting project https://github.com/io12/org-fragtog which I think can be integrated into this one. That would be convenient. Also will you consider to use a pop-up window for the preview? I see a package org-latex-impatient https://github.com/yangsheng6810/org-latex-impatient using pop-up window but the preview there is slow.

**Comment by guanyilun** [comment]
Yes I'm aware of org-fragtog and frequently use it with org-elp. I have tried implementing a pop-up window approach for the preview but stopped by some technical difficulty with adjusting the size of the pop-up window depending on the size of your preview. If I have time I will give it another go, but if you have some good ideas on how to implement it, that's also welcomed!
